conda_smithy/configure_feedstock.py

```diff
@@ -1741,9 +1751,10 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         "build_with_mambabuild": True,
         # feedstock checkout git clone depth, None means keep default, 0 means no limit
         "clone_depth": None,
-        # Specific channel for package can be given with
-        # `${url or channel_alias}::package_name`, defaults to conda-forge channel_alias
-        "remote_ci_setup": "conda-forge-ci-setup=3",
+        "remote_ci_setup": ["conda-forge-ci-setup=3"],
+        # Specific channel for packages for conda-forge-ci-setup,
+        # defaults to conda-forge
+        "remote_ci_setup_channels": ["conda-forge"],
```

Comment:
Let's remove this as it's not needed.
```suggestion
```

Reply:
Are you suggesting the user uses `remote_ci_setup` and adds the channels in there?

Seems a little difficult to get right with the escpae characters `<>|`

Reply:
Thank you for being so explicit! Thank you. Trying to simplify now. WIll see how the test goes.